django/utils/encoding.py

```diff
@@ -132,18 +132,9 @@ def force_bytes(s, encoding='utf-8', strings_only=False, errors='strict'):
         return s.encode(encoding, errors)
 
 
-smart_str = smart_text
-force_str = force_text
-
-smart_str.__doc__ = """
-Apply smart_text in Python 3 and smart_bytes in Python 2.
-
-This is suitable for writing to sys.stdout (for instance).
-"""
-
-force_str.__doc__ = """
-Apply force_text in Python 3 and force_bytes in Python 2.
-"""
+# Aliases for backwards-compatibility.
+smart_str = smart_str
+force_str = force_str
```

Comment:
Good catch, I forgot that when I re-did the PR after it conflicted with other changes. I'll amend and force push the change.